Cargo.toml

```diff
@@ -386,10 +386,12 @@ dispatch-releases = true
 github-release = "announce"
 # Whether to auto-include files like READMEs, LICENSEs, and CHANGELOGs (default true)
 auto-includes = false
+# Whether to put the binaries in the root of the archive or not, this only applies to unix archives
```

Comment:
Does that mean self-update on Windows does still not work?

Reply:
It was not a problem for the .zip files, there the files where always in the root of the file.